diff --git a/node_modules/@mytracker/react-native-mytracker/react-native-mytracker.podspec b/node_modules/@mytracker/react-native-mytracker/react-native-mytracker.podspec
index b1c5f33..f27cc6b 100644
--- a/node_modules/@mytracker/react-native-mytracker/react-native-mytracker.podspec
+++ b/node_modules/@mytracker/react-native-mytracker/react-native-mytracker.podspec
@@ -7,14 +7,14 @@ Pod::Spec.new do |s|
   s.version      = package["version"]
   s.summary      = package["description"]
   s.description  = package["description"]
-  s.homepage     = package["repository"]["baseUrl"]
+  s.homepage     = "https://getarent.ru"
 
   s.license      = package["license"]
 
   # s.license    = { :type => s.license, :file => package["licenseFilename"] }
   s.authors      = { package["author"]["name"] => package["author"]["email"] }
   s.platforms    = { :ios => "9.0" }
-  s.source       = { :git => package["repository"]["url"], :tag => "#{s.version}" }
+  s.source       = { :git => "https://getarent.ru", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,c,m,swift}"
   s.requires_arc = true
diff --git a/node_modules/@mytracker/react-native-mytracker/rn-mytracker/rn-mytracker.js b/node_modules/@mytracker/react-native-mytracker/rn-mytracker/rn-mytracker.js
index 3f80a4c..a8112bd 100644
--- a/node_modules/@mytracker/react-native-mytracker/rn-mytracker/rn-mytracker.js
+++ b/node_modules/@mytracker/react-native-mytracker/rn-mytracker/rn-mytracker.js
@@ -3,7 +3,7 @@ import { IS_IOS, IS_ANDROID, setBoolean, setString, setDict } from './rn-mytrack
 import { RNMyTrackerConfig } from './rn-mytracker-config.js';
 import { RNMyTrackerParams } from './rn-mytracker-params.js';
 import { StoreKitProduct, StoreKitTransaction } from './rn-mytracker-purchases.js';
-import { Platform } from 'react-native/types/index.js';
+import { Platform } from 'react-native';
 
 // Import platform-specific SDK API
 const { MyTracker } = NativeModules;
